Conversations 3 7 (#98)
tyntec/api-collection: conversations/v2/README.md

Version 2 of Conversations API will be discontinued on 31.12.2022. 

## 2.17

### Features

#### WhatsApp supporting Product and Product Lists

WhatsApp supports now sending products from a Facebook catalog.

This could be either a single product or multiple products via a product list.

> The message to be used is of type `interactive` with the respective subtype `product` or `productList`.

Reviewer: ```suggestion
The message type is `interactive` with the subtype `product` or `productList`.
```

> The catalog must be prior connected to the WhatsApp Business Account that shall have access to it.

Reviewer: ```suggestion
You need to link your Facebook Catalog to your WhatsApp Business Account. 
```

Reviewer: Do we have a tutorial for connecting the Catalog that we could link here?

> The customer can place an order via the app, in this case the new MoMessage event `MoMessage::Order` is triggered.

Reviewer: ```suggestion
The customer can place an order through their app. Then a new MoMessage event `MoMessage::Order` is triggered.
```
